soh/soh/Enhancements/custom-message/CustomMessageTypes.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 
 typedef enum {
+    TEXT_SSH = 0x22,
```

Comment:
minor: not super obvious what this is at first glance, don't need to change it but something to consider